### PR TITLE
Mac: converted compile instructions from RTF to markdown

### PR DESCRIPTION
**Pull request by neteler**
- changed since RTF format isn't really useful here.

Important: the instructions are probably outdated

### CODE REVIEW

**Comment by nilason** [comment]
> Important: the instructions are probably outdated

This is unfortunately a grave understatement. The issue #457 aim to address a lot of the problems with the present situation for mac and what's stated in this document.

I think it's good to convert the present text, even if it is outdated, for the simplification of git history research later on when this is updated. 

At the top may be inserted something like:
```
| :exclamation:  Instructions are at present not up-to-date. Update in progress. |
|-----------------------------------------|
```
which rendered looks like:
| :exclamation:  Instructions are at present not up-to-date. Update in progress.  |
|-----------------------------------------|

**Comment by nilason** [comment]
Some general markdown notes:
- header markdown should as with other md files in grass repo be noted with `#`'s
- title should be header `#`, not table

**Comment by cmbarton** [comment]
I'm happy  to replace it with my build instructions. But they use Anaconda for dependencies. Still, at least it works.

**Comment by nilason** [comment]
> I'm happy to replace it with my build instructions. But they use Anaconda for dependencies. Still, at least it works.

Good! I was also thinking of a short intro to how to build for macports. The dependencies are the tricky part for mac users (with them in place it is in practice only to do `configure; make; make install`). This in particular, now after years of being spoiled by W. Kyngesburye's great work on frameworks (which are still there but maybe not as frequently updated, at least not all of them, if I'm not mistaken). I believe homebrew/anaconda and macports are presently the best package systems for solving dependencies.

That being said, I think it would be good for historical purposes, to first convert the old instructions, and from there on update, modify. A deleted file, in this case the .rtf file, is always more difficult to trace after awhile.

**Comment by cmbarton** [comment]
Sounds good. I've got a big online workshop on open modeling to manage next week but will rework my Anaconda instructions for this after that. They are already in markdown as I am keeping them in a markdown notes app.

File: macosx/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,658 @@
+---
```
Comment (neteler) [inline macosx/ReadMe.md:1]: | :exclamation:  Instructions are at present not up-to-date. Update in progress. |
|--------------------------------------------------------------------------------|

File: macosx/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,658 @@
...
+title: GRASS Mac OS X Build
+---
```
Comment (neteler) [inline macosx/ReadMe.md:3]:
```suggestion
# title: GRASS GIS Mac OS X Build
```

File: macosx/ReadMe.md [hunk 3]
```diff
@@ -0,0 +1,656 @@
+# title: GRASS GIS Mac OS X Build
```
Comment (neteler) [inline macosx/ReadMe.md:1]: | :exclamation:  Instructions are at present not up-to-date. Update in progress. |
|--------------------------------------------------------------------------------|

**Comment by neteler** [comment]
ok, I'll merge now into master and then feel free to edit/replace/whatever needed.

**Pull request merged**